src/vm/amd64/asmconstants.h

```diff
@@ -663,7 +663,9 @@ ASMCONSTANTS_C_ASSERT(OFFSETOF__TransitionBlock__m_argumentRegisters == offsetof
 
 #undef ASMCONSTANTS_RUNTIME_ASSERT
 #undef ASMCONSTANTS_C_ASSERT
+#ifndef UNIX_AMD64_ABI
 #undef DBG_FRE
```

Comment:
/tmp/jithelpers_fast-77dca9.s:1040:32: error: unknown token in expression
        mov rcx, [r10 + DBG_FRE(0x38, 0x30)]
                               ^
make[2]: *** [src/vm/wks/CMakeFiles/cee_wks.dir/__/amd64/jithelpers_fast.S.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [src/vm/wks/CMakeFiles/cee_wks.dir/all] Error 2
make: *** [all] Error 2
Failed to build coreclr components.

I build it on Ubuntu 14.04.